app/src/main/java/edu/gatech/cs2340/hkskh/Shelters/Controllers/ShelterDetailActivity.java

```diff
@@ -42,9 +42,11 @@ protected void onCreate(Bundle savedInstanceState) {
         //Use passed hashcode from intent to gain access to correct shelter
         int shelterKey = getIntent().getIntExtra("shelter hash key", 0 );
         selected = (Shelter) (shelters.getShelter(shelterKey));
-        final String userName = this.getIntent().getStringExtra("Username");
+
+        //final String userName = this.getIntent().getStringExtra("Username");
         //Instantiate a UserManager and manage the check in and check out
         final edu.gatech.cs2340.hkskh.Users.UserManager users = new edu.gatech.cs2340.hkskh.Users.UserManager();
+        final String userName = users.getCurrUserName();
```

Comment:
We probably shouldn't be storing the current user in the `UserManager`, that isn't really its role. Ideally we'd use an `Application` class that stores the global state.